src/test/java/com/google/sps/data/QueryDatastoreTest.java

```diff
@@ -0,0 +1,81 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.sps.data.Comment;
+import com.google.sps.data.Constants;
+import com.google.sps.data.Parse;
+import com.google.sps.data.QueryDatastore;
+import com.google.sps.data.Post;
+import com.google.sps.data.Representative;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** */
+@RunWith(JUnit4.class)
+public final class QueryDatastoreTest {
+    private final LocalServiceTestHelper helper =
+        new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+    private DatastoreService ds; 
+
+    @Before
+    public void setUp() {
+        helper.setUp();
+        ds = DatastoreServiceFactory.getDatastoreService();
+    }
+
+    @After
+    public void tearDown() {
+        helper.tearDown();
+    }
+
+    @Test
+    public void testQueryForRepresentative() {
+        InsertAndUpdate.insertRepresentativeDatastore("Donald Trump", "President of the US");
+        Entity trumpEntity = QueryDatastore.queryForRepresentative("Donald Trump");
+        Assert.assertTrue(trumpEntity != null);
+        String name = (String) trumpEntity.getProperty(Constants.REP_NAME);
+        String title = (String) trumpEntity.getProperty(Constants.REP_TITLE);
+        Assert.assertTrue(name.equals("Donald Trump"));
+        Assert.assertTrue(title.equals("President of the US"));
+    }
```

Comment:
Might be good to separate by arrange, act, assert:

//Arrange 
InsertAndUpdate.insertRepresentativeDatastore("Donald Trump", "President of the US");

//Act
Entity trumpEntity = QueryDatastore.queryForRepresentative("Donald Trump");
String name = (String) trumpEntity.getProperty(Constants.REP_NAME);
String title = (String) trumpEntity.getProperty(Constants.REP_TITLE);

//Assert
Assert.assertTrue(trumpEntity != null);
Assert.assertTrue(name.equals("Donald Trump"));
Assert.assertTrue(title.equals("President of the US"));